source/code/plugins/filter_syslog.rb

```diff
@@ -30,6 +30,7 @@ def filter(tag, time, record)
       # Use Time.now, because it is the only way to get subsecond precision in version 0.12.
       # The time may be slightly in the future from the ingestion time.
       record["Timestamp"] = OMS::Common.format_time(Time.now.to_f)
+      record["EventTime"] = OMS::Common.format_time(time)
```

Comment:
I did some tests on my machine. It seems that FluentD parses the event time from syslog as being in UTC instead of in local time. The result is that, after formatting, 'EventTime' and 'TimeStamp' are a few hours apart.

For example:
"Timestamp":"2016-07-08T08:47:54.263Z",
"EventTime":"2016-07-08T15:32:45.000Z"